speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/model/RecognizeOptions.java

```diff
@@ -187,13 +187,13 @@ private Builder(RecognizeOptions recognizeOptions) {
       this.smartFormatting = recognizeOptions.smartFormatting;
       this.speakerLabels = recognizeOptions.speakerLabels;
       this.customizationId = recognizeOptions.customizationId;
-      this.interimResults = recognizeOptions.interimResults;
       this.grammarName = recognizeOptions.grammarName;
       this.redaction = recognizeOptions.redaction;
-      this.processingMetrics = recognizeOptions.processingMetrics;
-      this.processingMetricsInterval = recognizeOptions.processingMetricsInterval;
       this.audioMetrics = recognizeOptions.audioMetrics;
       this.contentType = recognizeOptions.contentType;
+      this.interimResults = recognizeOptions.interimResults;
+      this.processingMetrics = recognizeOptions.processingMetrics;
+      this.processingMetricsInterval = recognizeOptions.processingMetricsInterval;
```

Comment:
Is this model for websockets or for Recongnize Async? We shouldnt have `processingMetrics` and `processingMetricsInterval` in async

Reply:
This options model is shared by both the regular `recognize()` and the one using WebSockets, but it only applies to the WebSocket implementation.